extras/rails/ASSETS-README.md

```diff
@@ -1,17 +1,17 @@
-# Gulp Starter on Rails
+# Blendid on Rails
```

Comment:
should this section mention that it's probably a good idea to use [webpacker](https://github.com/rails/webpacker) instead of this, now that webpacker exists?

Reply:
I'm keeping this PR narrowly focused on the word "gulp", but yes there's definitely room to for future improvement